src/Collection.ts

```diff
@@ -304,6 +305,36 @@ export function createCollectionMixin<TBase extends Constructor>(Base: TBase) {
 
       return newIdx;
     }
+
+    /**
```

Comment:
Given a boundingbox, return all the objects of the collection that are contained in the box.
If includeInterseting is true, also the objects that are across the bounding box are selected.

Reply:
what about naming?

Reply:
i Don't have an opinion here. collectObjects if there is an explanation written on top is good enough.

Reply:
i thought it was ready. I will re-review.